### PR TITLE
Publish Dart doc

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we configure generation and publishing of Dart doc to [spine.io](https://spine.io).

In this PR we also resolve some issues with `pub`, publishing, such as putting `LICENSE` files in the published modules, updating the `pubspec.yml` to the new format, etc.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: client/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (armiol) [inline client/LICENSE:1]: Why do we have two license files instead of a single file in the repository root?

File: codegen/pubspec.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 name: dart_code_gen
 description: A command line tool which generates Dart code for Protobuf type registries.
-version: 0.1.0
+version: 0.1.1
```
Comment (armiol) [inline codegen/pubspec.yaml]: Do we have a different versioning policy of this module compared to the version of the library (which is `0.1.3` AFAIU)?

**Review by armiol — APPROVED** [review]
@dmdashenkov LGTM, but please see two questions before merging.

**Pull request merged**